MCAntiMalware-Core/src/main/java/optic_fusion1/antimalware/database/Database.java

```diff
@@ -45,8 +46,8 @@ public void downloadAndConnect() {
       try {
         Files.copy(new URL(DATABASE_URL).openStream(), tempFile.toPath(), StandardCopyOption.REPLACE_EXISTING);
       } catch (IOException e) {
-        Files.copy(Database.class.getResource("/database.db").openStream(), tempFile.toPath(),
-                StandardCopyOption.REPLACE_EXISTING);
+        Files.copy(Objects.requireNonNull(Database.class.getResource("/database.db")).openStream(), tempFile.toPath(),
```

Comment:
This one's not needed actually. It'll throw an error if it doesn't exist either way, however it'll just be an IOException.